soh/soh/Enhancements/randomizer/3drando/hint_list/hint_list_item.cpp

```diff
@@ -1952,6 +1952,10 @@ void HintTable_Init_Item() {
                        //clear text
                        Text{"Epona", /*french*/"Epona", /*spanish*/"a Epona"}
     );
+    hintTable[RHT_HINT_MYSTERIOUS] = HintText::Item({
```

Comment:
What is this used for?

Reply:
It's used for when we have Merchant shuffle on (No Hints) to make them say "mysterious item" instead of the actual item they sell.

Reply:
Was this the only question or was there something else in contention about this part?

Reply:
It was just a question as I couldn't think of anything that used it.

Reply:
No worries just checking if I could mark resolved!